tools/cli/internal/openapi/openapi.go

```diff
@@ -28,21 +28,10 @@ type Parser interface {
 }
 
 type Merger interface {
-	MergeOpenAPISpecs([]string) (*Spec, error)
+	MergeOpenAPISpecs([]string) (*openapi3.T, error)
 }
 
-type Spec struct {
-	OpenAPI      string                        `json:"openapi" yaml:"openapi"`
-	Security     openapi3.SecurityRequirements `json:"security,omitempty" yaml:"security,omitempty"`
-	Servers      openapi3.Servers              `json:"servers,omitempty" yaml:"servers,omitempty"`
-	Tags         openapi3.Tags                 `json:"tags,omitempty" yaml:"tags,omitempty"`
-	Info         *openapi3.Info                `json:"info" yaml:"info"`
-	Paths        *openapi3.Paths               `json:"paths" yaml:"paths"`
-	Components   *openapi3.Components          `json:"components,omitempty" yaml:"components,omitempty"`
-	ExternalDocs *openapi3.ExternalDocs        `json:"externalDocs,omitempty" yaml:"externalDocs,omitempty"`
-}
-
-func (o *OasDiff) MergeOpenAPISpecs(paths []string) (*Spec, error) {
```

Comment:
we need this to have the preserve the order of the fields similar to what we have right now in the MMS oas

Reply:
wooo TIL

Reply:
maybe you could add a comment above the struct so that the next time we don't forget about it

Reply:
on it